script/release-workflow/docker-build.sh

```diff
@@ -4,4 +4,4 @@ set -euo >/dev/null
 
 DOCKER_TARGET_PLATFORM=${DOCKER_TARGET_PLATFORM:-"linux/amd64"}
 
-docker buildx build --platform=${DOCKER_TARGET_PLATFORM} -t ${DOCKER_IMAGE_ORG_AND_NAME}:latest .
+docker buildx build --platform=${DOCKER_TARGET_PLATFORM} -t ${DOCKER_IMAGE_ORG_AND_NAME}:latest . --load
```

Comment:
Adds the `--load` command, due to this warning, which means the actual built image isn't actually scanned (at least on a local machine)

```
WARNING: No output specified with docker-container driver. Build result will only remain in the build cache. To push result image into registry use --push or to load image into docker use --load
```